packages/nimble-components/src/table/template.ts

```diff
@@ -38,95 +38,112 @@ export const template = html<Table>`
-                                    ?first-sorted-column="${(x, c) => x === c.parent.firstSortedColumn}"
-                                    @click="${(x, c) => c.parent.toggleColumnSort(x, (c.event as MouseEvent).shiftKey)}"
-                                    :isGrouped=${x => (typeof x.columnInternals.groupIndex === 'number' && !x.columnInternals.groupingDisabled)}
-                                >
-                                    <slot name="${x => x.slot}"></slot>
-                                </${tableHeaderTag}>
-                            `)}
-                        `)}
-                        <div class="header-scrollbar-spacer"></div>
-                    </span>
-                </div>
-            </div>
-            <div class="table-viewport" ${ref('viewport')}>
-                <div class="table-scroll"></div>
-                <div class="table-row-container" ${children({ property: 'rowElements', filter: elements(tableRowTag) })}
-                     role="rowgroup">
-                    ${when(x => x.columns.length > 0 && x.canRenderRows, html<Table>`
-                        ${repeat(x => x.virtualizer.visibleItems, html<VirtualItem, Table>`
-                            ${when((x, c) => (c.parent as Table).tableData[x.index]?.isGrouped, html<VirtualItem, Table>`
-                                <${tableGroupRowTag}
-                                    class="group-row"
-                                    :groupRowValue="${(x, c) => c.parent.tableData[x.index]?.groupRowValue}"
-                                    ?expanded="${(x, c) => c.parent.tableData[x.index]?.isExpanded}"
-                                    :nestingLevel="${(x, c) => c.parent.tableData[x.index]?.nestingLevel}"
-                                    :leafItemCount="${(x, c) => c.parent.tableData[x.index]?.leafItemCount}"
-                                    :groupColumn="${(x, c) => c.parent.tableData[x.index]?.groupColumn}"
-                                    ?selectable="${(_, c) => c.parent.selectionMode === TableRowSelectionMode.multiple}"
-                                    selection-state="${(x, c) => c.parent.tableData[x.index]?.selectionState}"
-                                    @group-selection-toggle="${(x, c) => c.parent.onRowSelectionToggle(x.index, c.event as CustomEvent<TableRowSelectionToggleEventDetail>)}"
-                                    @group-expand-toggle="${(x, c) => c.parent.handleGroupRowExpanded(x.index, c.event)}"
-                                >
-                                </${tableGroupRowTag}>
+            <div class="glass-pane" style="
+                --ni-private-glass-pane-pointer-events: ${x => (x.isColumnBeingSized ? 'none' : 'default')};
+            ">
+                <div role="rowgroup" class="header-row-container">
+                    <div class="header-row" role="row">
+                        <span class="header-row-action-container" ${ref('headerRowActionContainer')}>
```

Comment:
I initially added this wrapper element (and provided it a `ref`) because at one point my implementation needed to more easily track when the size of the combination of the contained elements changed. While this is no longer the case, its presence does clean up a pageObject implementation, so I'm not sure if that is enough to keep this around.